src/app/components/SummaryHighlight/SummaryHighlight.tsx

```diff
@@ -0,0 +1,40 @@
+import classes from './SummaryHighlight.module.css';
+
+type Options = {
+  hour: 'numeric';
+  minute: 'numeric';
+  second: 'numeric';
+  hour12: boolean;
+  timeZone: string;
+};
+
+function createTimestampOutput(timestamp: string): string {
+  const date = new Date(Number(timestamp));
+  const options: Options = {
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: false,
+    timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+  };
+  return new Intl.DateTimeFormat('default', options).format(date);
```

Comment:
I thought we like to display the seconds from the beginning of the match instead of the datetime?